src/microbe_stage/TweakedProcess.cs

```diff
@@ -5,12 +5,14 @@
 /// </summary>
 public class TweakedProcess : ICloneable
 {
+    public float Count;
```

Comment:
If the count is kept here this *must be* documented as it's very confusing to have both a count and the actual rate. There needs to be comments explaining what's the difference here.